components/expansion-panel/readme.md

```diff
@@ -13,6 +13,7 @@
 | `noRipple`         | `noRipple`         | `boolean`         | false     |                                                                                            |
 | `open`             | `open`             | `boolean`         | false     |                                                                                            |
 | `ripple`           |                    | `Promise<Ripple>` |           |                                                                                            |
+| `heading-level`     | `3`                | `string` | "3"       | The expension panel headline - default is `H3` but need to set according to the page hierarchy      |
```

Comment:
Attribute is still wrong

Reply:
the attribute is OK the description isn't. Fixed it.